authors/apps/authentication/serializers.py

```diff
@@ -10,10 +10,53 @@ class RegistrationSerializer(serializers.ModelSerializer):
 
     # Ensure passwords are at least 8 characters long, no longer than 128
     # characters, and can not be read by the client.
-    password = serializers.CharField(
-        max_length=128,
+    # serializers.RegexField - A text representation, that validates the given value matches against a certain regular expression.
+
+    password = serializers.RegexField(
+        regex="^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,}$",
         min_length=8,
-        write_only=True
+        max_length=128,
```

Comment:
@Hesbon5600 128 is alot, reset this to 30 max characters.

Reply:
Okay sure